Makefile

```diff
@@ -26,9 +26,10 @@ EXTERNAL_TOOLS := \
 all: build
 
 git_chglog_check:
-	if [ -z "$(shell which git-chglog)" ]; then \
-		go get -u -v github.com/git-chglog/git-chglog/cmd/git-chglog && git-chglog --version; \
-	fi
+ifeq ($(shell which git-chglog),)
+	go get -u -v github.com/git-chglog/git-chglog/cmd/git-chglog
```

Comment:
Future: Since this PR is just a syntax change, no need to tackle it here. But `go get -u` is going to update the `go.mod`/`go.sum` file, which is not something we want for installing executables.

If we require go 1.16 for development and bump our pipeline, we can switch to `go install <url>@latest`, and if not, it's probably better to run these with `GO111MODULE=off`.